src/source/PeerConnection/Rtp.c

```diff
@@ -136,27 +136,43 @@ STATUS transceiverOnBandwidthEstimation(PRtcRtpTransceiver pRtcRtpTransceiver, U
     return retStatus;
 }
 
+STATUS transceiverOnPictureLoss(PRtcRtpTransceiver pRtcRtpTransceiver, UINT64 customData, RtcOnPictureLoss onPictureLoss)
```

Comment:
How is this related to the PR?

Reply:
i refactored handling of PLI packets (`transceiverOnPictureLoss` is public API for PLI callback)
to be absolutely honest it is not really related to this PR :) 
just added for completeness and because i needed it in production

Reply:
want me to take it out?

Reply:
Cool! It would be super helpful if you could create a new PR with the onPictureLoss changes. It is just to make sure that related changes go into a single commit at the end so that it is independently tested. Thats all :)

Reply:
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/692